tests/functional/odf-cli/test_get_commands.py

```diff
@@ -1,24 +1,45 @@
 import logging
 import re
+import pytest
 
 from ocs_ci.ocs.resources.pod import get_mon_pods
 from ocs_ci.helpers.odf_cli import ODFCLIRetriever, ODFCliRunner
-from ocs_ci.framework.testlib import tier1, brown_squad, polarion_id
+from ocs_ci.framework.testlib import tier1, brown_squad, polarion_id, skipif_ocs_version
 
-logger = logging.getLogger(__name__)
+log = logging.getLogger(__name__)
 
 
 @tier1
 @brown_squad
+@skipif_ocs_version("<4.15")
 class TestGetCommands:
-    def test_download_binary(self):
-        self.odf_cli_retriever = ODFCLIRetriever()
-        self.odf_cli_retriever.retrieve_odf_cli_binary()
-        self.odf_cli_runner = ODFCliRunner()
+    @pytest.fixture(scope="function", autouse=True)
+    def odf_cli_setup(self):
```

Comment:
shouldn't ODF CLI binary be validated as part of post deployment or maybe as a session scope fixture?

Reply:
im not sure about post deployment, but it can be session scope. i will test it as session